webapp/components/notifications/NotificationMenu/NotificationMenu.vue

```diff
@@ -50,32 +50,20 @@ export default {
   },
   watch: {
     notifications: {
+      immediate: true,
       handler(lastQueriedNotifications) {
         if (lastQueriedNotifications && lastQueriedNotifications.length > 0) {
-          // set this to be empty to get always called if a query comes with results from the backend
-          // has the sideeffect the handler is encouraged to be called again, but only once with no effect, because of the if above
-          this.notifications = []
```

Comment:
Why you removed this line of code? After our discussion …

@mattwr18 You and I agreed yesterday to leave this line in the code with a specific comment, that I found out that this was needed for a resolution of a misfunctioning I saw, but that I can not exactly say at the moment what this was and we both can not find out right now.

Please don’t do that without talking to me again! 🙏🏼

Reply:
It wasn't done on purpose and I have no idea how it was deleted... maybe I forgot to save the file before pushing up.
As I didn't do it mindfully, I cannot guarantee that I will not do it again, such it is in life.
Thanks for your understanding

Reply:
Okay, I understand.